VOKMockUrlProtocol.m

```diff
@@ -155,6 +166,10 @@ - (NSArray *)resourceNames
                 [resourceNames addObject:[[name stringByAppendingFormat:AppendSeparatorFormat, bodyString] mutableCopy]];
             }
             [resourceNames addObject:[[name stringByAppendingFormat:AppendSeparatorFormat, bodyHash] mutableCopy]];
+            
+            if (AllowsWildcard) {
+                [resourceNames addObject:[[name stringByAppendingFormat:AppendSeparatorFormat, Wildcard] mutableCopy]];
+            }
         }
     }
```

Comment:
I think, at this point, the ordered list of possible names is something like:
1. `…|queryString|bodyString`
1. `…|queryString|bodyHash`
1. `…|queryString|*`
1. `…|queryHash|bodyString`
1. `…|queryHash|bodyHash`
1. `…|queryHash|*`
1. `…|*|bodyString`
1. `…|*|bodyHash`
1. `…|*|*`

It seems to me like this might allow a wildcard resource to have higher precedence than a non-wildcard resource, if one mixes query strings and query hashes, but I'm not entirely sure.